packages/server/lib/browsers/chrome.ts

```diff
@@ -55,6 +55,18 @@ let browserCriClient: BrowserCriClient | undefined
  * @param userDir
  */
 const _getChromePreferences = (userDir: string): Bluebird<ChromePreferences> => {
+  /**
+   * By returning empty objects here, the user is indicating that they do not want
+   * Chrome preferences read. If they do not provide preferences via config,
+   * this effectively prevents Cypress from writing to the Chrome preferences files.
+   * See: _writeChromePreferences
+   */
+  if (process.env.IGNORE_CHROME_PREFERENCES) {
+    debug('ignoring chrome preferences...')
+
+    return Bluebird.resolve(_.mapValues(CHROME_PREFERENCE_PATHS, () => ({})))
+  }
+
   debug('reading chrome preferences... %o', { userDir, CHROME_PREFERENCE_PATHS })
 
   return Bluebird.props(_.mapValues(CHROME_PREFERENCE_PATHS, (prefPath) => {
```

Comment:
We are already catching the `ENOENT` error below and returning an empty object. Is there an error thrown when reading an encrypted path? If so, can we just add it to below and remove the need of the new environment variable?

Reply:
It might be the write logic that throws the error, not the read?